### PR TITLE
Feature/maji 3 icon fix

### PR DESCRIPTION
**Pull request by RoelN**
Prevent ID clashes and move icons out of style folder, into assets folder.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
   },
   "devDependencies": {
     "chai-dom": "^1.4.3",
+    "file-loader": "^0.11.2",
```
Comment (pascalw) [inline package.json]: I think you meant to add these modules to the project template package.json, this is the maji module package.json.

**Comment by pascalw** [comment]
@RoelN icons in the example app are broken with this change:

screen shot 2017-09-01 at 07 41 16

This is probably because you [removed passing all props to icons](https://github.com/kabisa/maji/pull/159/files#diff-cd5e19a2ad84cdecf0822efc5a224f4aL5) but this feature is used to pass a class name [over here](https://github.com/kabisa/maji/blob/feature/maji-3/project_template/src/components/NavList.js#L15).

Would it suffice to omit the `id` property from the props?

**Comment by RoelN** [comment]
Ah good one, apologies for not catching that. I've moved the type of icon you want to a custom attibute so it looks like this: `<Icon icon="back">` because I found `id` not really fitting anyway. Would that work for you folks?

Also, don't we need the `url-loader` as a non-dev dependency? 🤔

**Comment by pascalw** [comment]
@RoelN `<Icon icon="back"/>` is fine by me. We could also do `<Icon back />` but that would be a little trickier because there's no straightforward way to differentiate the identifier vs other props.

**Comment by pascalw** [comment]
Oh and no we don't need `url-loader` in the root `package.json`. Maji itself doesn't even use Webpack, but rollup.

**Pull request merged**